### PR TITLE
Add fix for testing as a process with CA signed certificate flow

### PR DESCRIPTION
**Pull request by sigalsax**
### What does this PR do?
Some tests were failing as a result of working with a DAP master that has a CA signed cert. This PR fixes this

### CODE REVIEW

File: test/test_integration_credentials.py [hunk 1]
```diff
@@ -215,37 +193,26 @@ def test_https_netrc_was_not_overwritten_when_login_failed_but_already_logged_in
...
-            'Authorization': f'Token token="{access_token}"'
-        }
-        url = f"{self.client_params.hostname}/authn/{self.client_params.account}/api_key?role=host:somehost"
-        # We want to rotate the API key of the host so we will know the value
-        host_api_key = requests.request("PUT", url, headers=headers, data=self.client_params.env_api_key,
-                                        verify=self.environment.path_provider.certificate_path).text
+        extract_api_key_from_message = host_api_key.split(":")[1].strip()
 
-        os.remove(DEFAULT_NETRC_FILE)
+        self.invoke_cli(self.cli_auth_params,
```
Comment (tzheleznyak) [inline test/test_integration_credentials.py:209]: This can be one line

File: test/test_integration_credentials.py [hunk 2]
```diff
@@ -102,43 +94,29 @@ def test_https_netrc_is_created_with_all_parameters_given(self, mock_input):
...
-        requests.request("PUT", f"{self.client_params.hostname}/authn/{self.client_params.account}/password",
-                         headers=headers, data=password,
-                         verify=self.environment.path_provider.certificate_path)
-
-        # Need to remove the netrc because we are attempting to login as a user with their new password
-        os.remove(DEFAULT_NETRC_FILE)
+        self.invoke_cli(self.cli_auth_params,
+                        ['user', 'change-password', '-p', password])
+
+        self.invoke_cli(self.cli_auth_params,
```
Comment (tzheleznyak) [inline test/test_integration_credentials.py:111]: make this one line

Comment (sigalsax) [reply test/test_integration_credentials.py:111]: i think its clearer if it isnt

**Review by tzheleznyak — DISMISSED** [review]

**Review by tzheleznyak — APPROVED** [review]
LGTM

**Pull request merged**